helm-org.el

```diff
@@ -273,6 +273,73 @@ current heading."
         :truncate-lines helm-org-truncate-lines
         :buffer "*helm org capture templates*"))
 
+;;; Org tag completion
+
+;; Based on code from Anders Johansson posted on 3 Mar 2016 at
+;; <https://groups.google.com/d/msg/emacs-helm/tA6cn6TUdRY/G1S3TIdzBwAJ>
+
+;; FIXME: This function fails with a "Wrong number of arguments" error
+;; if its name starts with "helm"?!  If its name starts with anything
+;; else, it works fine!
+(defun not-helm-org-completing-read-tags (prompt coll pred req initial hist def inh)
+  (if (not (string= "Tags: " prompt))
+      ;; Not a tags prompt.  Use normal completion by calling
+      ;; `org-icompleting-read' again without this function in
+      ;; `helm-completing-read-handlers-alist'
+      (let ((helm-completing-read-handlers-alist (rassq-delete-all
```

Comment:
I using  [Emacs-mac](https://bitbucket.org/mituharu/emacs-mac) with [Spacemacs](https://github.com/syl20bnr/spacemacs).


**(emacs-version)**:
GNU Emacs 25.2.1 (x86_64-apple-darwin16.6.0, Carbon Version 157 AppKit 1504.83)

**org-version**:
9.0.7

**helm**:
Latest; I update packages daily.
(current: elpa/helm-20170520.1308)

---
`rassq-delete-al` in My Emacs:
```emacs-lisp
(defun rassq-delete-all (value alist)
  "Delete from ALIST all elements whose cdr is `eq' to VALUE.
Return the modified alist.
Elements of ALIST that are not conses are ignored."
  (while (and (consp (car alist))
              (eq (cdr (car alist)) value))
    (setq alist (cdr alist)))
  (let ((tail alist) tail-cdr)
    (while (setq tail-cdr (cdr tail))
      (if (and (consp (car tail-cdr))
               (eq (cdr (car tail-cdr)) value))
          (setcdr tail (cdr tail-cdr))
        (setq tail tail-cdr))))
  alist)
```